### DISCUSSION TITLE
Change "Person detection" to be always Enable in camera stock firmware - IFUS - 11.0.36.7.xxx

### DISCUSSION THREAD

**Discussion by peterek1983** [roleoroleo/yi-hack-Allwinner-v2#423] [Q&A]
I have IFUS 11.0.36.7.xx stock camera firmware without hack. I'd like only change one setting to be always Enable after power on/off camera I mean AI Person detection.  
I noticed with newest Yi Home app this is not possible to enable anymore with latest 11.0.36.7xx firmware if you are not Kami Cloud subscriber, but I can enable Person detection in older Yi Home app and it work. But after power off camera this Enabled option is still Disabled.  
Does it possible to Enable it to be always active after power toggle camera ?

I wish to downgrade this 11.0.36.7 firmware to previous one 9.0.36 but really do not know how to do this or is it possible.

**Comment by roleoroleo** [comment]
I'm not sure because I don't own this cam.
But I think that human detection is moved from local to cloud between version 9 and 11.
I think it's not possible to enable human detection locally.

**Comment by peterek1983** [comment]
thank you roleroleo for answer. 

How do you think is it possible to downgrade to version 9 in some way ?

**Comment by roleoroleo** [comment]
Check this post: https://github.com/roleoroleo/yi-hack-Allwinner-v2/issues/388
You need partition files of version 9.
Put them in the sd card with the correct name:
| Partition | Name |
| --- | --- |
| mtd2 | rootfs_y211ga |
| mtd3 | home_y211ga |
| mtd4 | backup_y211ga |

Switch on and cross your fingers.
